config/motoros2_config.yaml

```diff
@@ -284,3 +284,35 @@ publisher_qos:
 #
 # DEFAULT: false
 #ignore_missing_calib_data: false
+
+#-----------------------------------------------------------------------------
+# Should MotoROS2 broadcast debug messages?
+#
+# If enabled, this will broadcast log messages on the network on port UDP 21789.
+# the user can use the debug script provided in the distribution of MotoROS2 to
+# monitor the state of the robot, identify problems, and debug their code.
+#
+# The debug script is available under the Yaskawa-Global/motoros2 repository in
+# the tools directory
+# https://github.com/Yaskawa-Global/motoros2/tree/main/tools
+#
+# DEFAULT: true
+#userlan_debug_broadcast_enabled: true
+
+#-----------------------------------------------------------------------------
+# Which network port should MotoROS2 broadcast the debug messages to, if
+# 'userlan_debug_broadcast_enabled' is 'true'?
+#
+# If not specified and 'userlan_debug_broadcast_enabled' is true, MotoROS2 will
+# send the broadcast to all network ports which are enabled on the controller.
+#
+# To choose a specific port to broadcast debug messages, uncomment
+# 'userlan_debug_broadcast_port' below and set it to the desired port.
+#
+# NOTE 1:   this setting only applies to YRC1000 and YRC1000u controllers.
+#         DX200 and FS100 controllers only have a single ethernet port, and will
+#         always default to USER_LAN1
+#
+# OPTIONS: USER_LAN1, USER_LAN2
+# DEFAULT: (all available network ports)
+#userlan_debug_broadcast_port: USER_LAN1
```

Comment:
what would be the value (ie: string) for 'all available network ports'? Because the documentation states that's the default, but I can't set this to `'all available network ports'` I believe.

Reply:
So there is not a dedicated string for 'all available ports'. I was planning on having that, but part of the weirdness with it having the same type as `userlan_monitor_port` (`Ros_UserLan_Port_Setting`) is that even if they have different options with different behavior (e.g. `userlan_monitor_port` has 'auto detection' and `userlan_debug_broadcast_port` has 'all available ports'), they are parsed the same. So I had the behavior match that of `userlan_monitor_port`, which doesn't have a specific string for 'auto detection'. That being said, unlike `userlan_monitor_port`, if the user types in an invalid string, it will default to 'all available ports' rather than 'disabled'